src/scss/main.scss

```diff
@@ -79,4 +160,346 @@ strong {
 @import "sections/header";
 
 // Import components
-@import "components/button";
\ No newline at end of file
+@import "components/button";
+
+// Templates
+
+%tpltitle {}
+
+%tplbutton {
+  padding: 13px 61px 15px 36px;
+  text-decoration: none;
+  line-height: 26px;
+  position: relative;
+  color: #fff;
+  background-color: #0E0E0E;
+}
+
+// Global settings
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-style: $default-font;
+  font-size: $deafault-font-size;
+  font-weight: $deafault-font-weight;
+  color: $text;
+  background-color: $backgorund;
+}
+
+// Header
+.container {
+  padding: 0 98px;
+  max-width: 100%;
+}
+
+.intro {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 26px 0 117px;
+  position: relative;
+
+  &_name {
+    span {
+      font-size: 20px;
+    }
+  }
+
+  &_menu {
+    display: none;
+  }
+}
+
+.intro_info {
```

Comment:
Refactor to `&_info` and move to the top section

Reply:
Changed